src/aerospike_vector_search/aio/admin.py

```diff
@@ -176,6 +176,60 @@ async def index_create(
             logger.error("Failed waiting for creation with error: %s", e)
             raise types.AVSServerError(rpc_error=e)
 
+
+    async def index_update(
+        self,
+        *,
+        namespace: str,
+        name: str,
+        index_labels: Optional[dict[str, str]] = None,
+        hnsw_update_params: Optional[types.HnswIndexUpdate] = None,
+        timeout: Optional[int] = 100_000,
+    ) -> None:
+        """
+        Update an existing index.
+
+        :param namespace: The namespace for the index.
+        :type namespace: str
+
+        :param name: The name of the index.
+        :type name: str
+
+        :param index_labels: Optional labels associated with the index. Defaults to None.
+        :type index_labels: Optional[dict[str, str]]
+
+        :param hnsw_update_params: Parameters for updating HNSW index settings.
+        :type hnsw_update_params: Optional[types.HnswIndexUpdate]
+
+        :param timeout: Timeout  in seconds for internal index update tasks. Defaults to 100_000.
+        :type timeout: int
+
+        Raises:
+            AVSServerError: Raised if an error occurs during the RPC communication with the server while attempting to update the index.
+        """
+
+        await self._channel_provider._is_ready()
+
+        (index_stub, index_update_request, kwargs) = self._prepare_index_update(
+            namespace= namespace,
+            name =name,
+            index_labels= index_labels,
+            hnsw_update_params= hnsw_update_params,
+            logger = logger,
+            timeout= timeout
```

Comment:
```suggestion
            namespace = namespace,
            name = name,
            index_labels = index_labels,
            hnsw_update_params = hnsw_update_params,
            logger = logger,
            timeout = timeout
```
formatting is strange here